### PR TITLE
change Windows App Runtime version to actually reflect runtime, not SDK

### PR DESCRIPTION
**Pull request by Scottj1s**
Note: Microsoft.WindowsAppRuntime.Insights.Resource.dll is the only module that carries complete version info.

### CODE REVIEW

File: WinUIGallery/App.xaml.cs [hunk 1]
```diff
@@ -42,15 +36,20 @@ public static string WinAppSdkDetails
...
+        }
+
         public static string WinAppSdkRuntimeDetails
         {
             get
-            {
-                var details = WinAppSdkDetails;
+            {
+                var details = WinAppSdkDetails;
 #if WindowsAppSdkRuntimeDependent
```
Comment (codendone) [inline WinUIGallery/App.xaml.cs:46]: >WindowsAppSdkRuntimeDependent [](http://example.com/codeflow?start=4&length=29)

With the new implementation, is there any need to condition this?

Comment (Scottj1s) [reply WinUIGallery/App.xaml.cs:46]: well no, come to think of it

**Review by codendone — APPROVED** [review]
:shipit:

**Comment by Jay-o-Way** [comment]
Question: what if there is an issue with something like `windowsAppRuntimeVersion.First()`? Like, empty collection or something like that...

**Comment by Scottj1s** [comment]
> Question: what if there is an issue with something like `windowsAppRuntimeVersion.First()`? Like, empty collection or something like that...

First() will throw InvalidOperationException if the collection is empty.  This is not expected here.

**Comment by Jay-o-Way** [comment]
@Scottj1s I was looking at #1528 at the time. #1480 holds the solution.

**Comment by Scottj1s** [comment]
ah, thanks for the explanation.  I'll follow up on https://github.com/microsoft/WinUI-Gallery/pull/1480.

**Pull request merged**